src/main/kotlin/dsl/Languages.kt

```diff
@@ -0,0 +1,13 @@
+package dsl
+class Language(val language: String, val level: Int)
+@JvmInline
+value class Languages(val values: List<Language>)
+class LanguagesBuilder {
+    private val languages = mutableListOf<Language>()
+    infix fun String.level(level: Int) {
+        languages.add(Language(this, level))
+    }
+    fun build(): Languages {
+        return Languages(languages)
+    }
+}
```

Comment:
```suggestion
class Language(val language: String, val level: Int)

@JvmInline
value class Languages(val values: List<Language>)

class LanguagesBuilder {
    private val languages = mutableListOf<Language>()
    
    infix fun String.level(level: Int) {
        languages.add(Language(this, level))
    }
    
    fun build(): Languages {
        return Languages(languages)
    }
}
```

개행 라인을 잘 활용한다면 가독성을 높일수 있습니다 😄